great_expectations/profiler/parameter_builder/multi_batch_bootstrapped_metric_distribution_parameter_builder.py

```diff
@@ -66,4 +83,7 @@ def _build_parameters(self, *, rule_state, validator, batch_ids, **kwargs):
                 )
             )
 
-        return {"parameters": samples[0]}
+        return Parameter(
+            parameters=samples[0],
```

Comment:
Just a suggestion to add a TODO for later
```suggestion
            # TODO: Using the first sample for now, but this should be extended for handling multiple batches
            parameters=samples[0],
```